test/WebJobs.Script.Tests/Eventing/SystemLoggerTests.cs

```diff
@@ -64,13 +64,17 @@ public void Trace_Verbose_LogData_EmitsExpectedEvent()
             string message = "TestMessage";
             string functionInvocationId = Guid.NewGuid().ToString();
             string activityId = Guid.NewGuid().ToString();
+            var scopeState = new Dictionary<string, object>
+            {
+                [ScriptConstants.LogPropertyFunctionInvocationIdKey] = functionInvocationId
+            };
+            DictionaryLoggerScope.Push(scopeState);
```

Comment:
Rather than pushing directly to the `DictionaryLoggerScope`, change this test to look like:

    using (_logger.BeginScope(scopeState))
    {
        _logger.Log(...);
    }

That will test the scenario the way it runs in production.